.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: rust
 sudo: false
 rust:
-  - stable
-  - beta
+#  - stable
+#- beta
   - nightly
-matrix:
-  allow_failures:
-    - rust: nightly
+#matrix:
+#  allow_failures:
+#    - rust: nightly
```

Comment:
It shows that it's only temporary. I hope to be able to use this on beta soon.

Should I still remove it?